### PR TITLE
Unify copyright, clarify contributing and code of conduct

### PR DESCRIPTION
**Pull request by white238**
Got advice from Charity to follow Spack's example.

### CODE REVIEW

File: SetupBLT.cmake [hunk 1]
```diff
@@ -1,44 +1,7 @@
-###############################################################################
-# Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC.
-#
-# Produced at the Lawrence Livermore National Laboratory
-#
-# LLNL-CODE-725085
```
Comment (kennyweiss) [inline SetupBLT.cmake]: @white238 and others -- could you please confirm that we no longer need to include ``LLNL-CODE-725085``?

Should it go in ``COPYRIGHT`` or ``README.md``?

Comment (white238) [reply SetupBLT.cmake]: It’s in the top level one.

Comment (kennyweiss) [reply SetupBLT.cmake]: Thanks @white238 -- I think I missed it because it wasn't modified in this PR.

**Comment by white238** [comment]
Thoughts on adding a section to the contributions file about how to attribute yourself on your changes?

New files add yourself as the author on the top

New macros add yourself as author in comment above..

Small changes don’t ...

?

**Comment by cyrush** [comment]
@white238 this looks great! Not sure about how to add contributors, other than adding names to the docs?  I think the git commits + PRs can be used for attribution.

**Pull request merged**